CODEOWNERS

```diff
@@ -1,18 +1,40 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-# These are owners for a specific engineering file or folder
-site/*  @monzo/web-approvers
-.eslintignore  @monzo/web-approvers
-.eslintrc  @monzo/web-approvers
-.flowconfig  @monzo/web-approvers
-.gitignore  @monzo/web-approvers
-.prettierrc  @monzo/web-approvers
-CODEOWNERS  @monzo/web-approvers
-package.json  @monzo/web-approvers
-yarn.lock  @monzo/web-approvers
-*.js  @monzo/web-approvers
-*.json  @monzo/web-approvers
-
 # These owners will be the default owners for everything in the repo.
-*       @lucasjohnston
\ No newline at end of file
+*       @lucasjohnston
```

Comment:
I wonder if we should have this going forward? It seems you'd be covered for most technical stuff by the web-approvers group?

Reply:
By being at the top, the * is effectively a last resort - it means this `CODEOWNER` will only be requested if the changes do not match any other rule (which should never happen) - I'll probably change this to web-approvers or the whole org in the future, but at the mo I think it's fine to be me

Reply:
So I won't be tagged in everything 😛